spec/models/list_import/anime_planet_spec.rb

```diff
@@ -0,0 +1,93 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: list_imports
+#
+#  id                      :integer          not null, primary key
+#  error_message           :text
+#  error_trace             :text
+#  input_file_content_type :string
+#  input_file_file_name    :string
+#  input_file_file_size    :integer
+#  input_file_updated_at   :datetime
+#  input_text              :text
+#  progress                :integer
+#  status                  :integer          default(0), not null
+#  strategy                :integer          not null
+#  total                   :integer
+#  type                    :string           not null
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#  user_id                 :integer          not null
+#
+# rubocop:enable Metrics/LineLength
+
+require 'rails_helper'
+
+RSpec.describe ListImport::AnimePlanet do
+  before do
+    host = described_class::ANIME_PLANET_HOST
+    extensions = '?mylist_view=grid&per_page=480&sort=title&page='
+
+    # Page 1 Toy
+    stub_request(:get, "#{host}toyhammered/anime#{extensions}1")
+      .to_return(body: fixture('list_import/anime_planet/toy-anime.html'))
+    stub_request(:get, "#{host}toyhammered/manga#{extensions}1")
+      .to_return(body: fixture('list_import/anime_planet/toy-manga.html'))
+
+    # Page 2 Toy
+    stub_request(:get, "#{host}toyhammered/anime#{extensions}2")
+      .to_return(body: fixture('list_import/anime_planet/toy-anime.html'))
+
+    # Page 1 Sierra
+    stub_request(:get, "#{host}sierrawithas/anime#{extensions}1")
+      .to_return(body: fixture('list_import/anime_planet/sierra-anime.html'))
+    stub_request(:get, "#{host}sierrawithas/manga#{extensions}1")
+      .to_return(body: fixture('list_import/anime_planet/sierra-manga.html'))
+  end
+
+  it { should validate_presence_of(:input_text) }
+  it { should validate_length_of(:input_text)
```

Comment:
>Prefer do...end for multi-line blocks without chaining. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

This can be ignored in specs